manifests/rhoai/monitor.yaml

```diff
@@ -0,0 +1,13 @@
+# Prometheus Pod Monitor (Metrics)
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
```

Comment:
Ideally, we would remove the Service created for the metrics endpoint from base. Kueue `rhoai` overlay does that for reference.

Reply:
Ok, i will add patch to delete the service from base.